tools/src/format.rs

```diff
@@ -102,7 +102,12 @@ pub fn format_code(jumps: &HashMap<String, Jump>, format: FormatAs,
 
         let data = match (&token.kind, datum) {
             (&tokenize::TokenKind::Identifier(None), Some(d)) => {
-                let ref id = d[0].sym;
+                let id = if d.len() == 1 {
+                    &d[0].sym
+                } else {
+                    let displayed = &input[token.start .. token.end];
+                    &d.iter().find(|ref item| { item.sym.contains(displayed) }).unwrap_or(&d[0]).sym
```

Comment:
nit: no need for `ref` (`find` already gives you a reference I'd think), nor for braces in single line closures (though I notice they're used below too). So also your call :)